### PR TITLE
Update research section

### PR DESCRIPTION
**Pull request by rrhyne**
Small update to the research section

### CODE REVIEW

File: handbook/product/design/design-process.md [hunk 1]
```diff
@@ -32,27 +32,26 @@ While asynchronous communication is a core attribute of remote work, key moments
...
+    - When updating existing product features, first conduct usability tests to inform redesign decisions
   - Process
       - A GitHub issue should be created to track the work
       - Research is conducted according to requirements
-      - Publish research to Google Drive
+      - Findings publishing and review
+        - Findings should be detailed in a google doc and announced in the corresponding team channel or #product on Slack
+        - Findings should contain a clear summary, links to resources and supporting data
+        - Conduct a syncronous review meeting with key stakeholders based on side of project or importance of findings
       - Announce research in Slack
```
Comment (AlicjaSuska) [inline handbook/product/design/design-process.md:48]: @rrhyne Good changes! I have only one suggestion here. It seems like announcing research in Slack is already mentioned in line 45. Maybe we can remove this line (48)? :)

Comment (rrhyne) [reply handbook/product/design/design-process.md]: Done!

File: handbook/product/design/design-process.md [hunk 2]
```diff
@@ -32,27 +32,25 @@ While asynchronous communication is a core attribute of remote work, key moments
...
+    - When updating existing product features, first conduct usability tests to inform redesign decisions
   - Process
       - A GitHub issue should be created to track the work
       - Research is conducted according to requirements
-      - Publish research to Google Drive
-      - Announce research in Slack
-      - Schedule and conduct a presentation meeting with stakeholders
-      - Stakeholders will approve the GitHub issue indicating the designs are ready for the next phase
+      - Findings publishing and review
+        - Findings should be detailed in a google doc and announced in the corresponding team channel or #product on Slack
```
Comment (christinaforney) [inline handbook/product/design/design-process.md:45]:
```suggestion
        - Findings should be detailed in a Google doc and announced in the corresponding team channel or #product on Slack
```

File: handbook/product/design/design-process.md [hunk 3]
```diff
@@ -32,27 +32,25 @@ While asynchronous communication is a core attribute of remote work, key moments
...
+        - Findings should contain a clear summary, links to resources and supporting data
+        - Conduct a syncronous review meeting with key stakeholders based on side of project or importance of findings
```
Comment (christinaforney) [inline handbook/product/design/design-process.md:47]:
```suggestion
        - Conduct a synchronous review meeting with key stakeholders based on the size of project or importance of findings
```

**Pull request merged**